### PR TITLE
Simplify the example

### PR DESCRIPTION
**Pull request by nromanetti**
Simpler exemple with an in memory key repo...
Doc needs to be adjusted (TODO)

### CODE REVIEW

File: samples/generatedkey-sample/pom.xml [hunk 1]
```diff
@@ -47,6 +47,11 @@
             <artifactId>kafka-encryption-generatedkey</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.quicksign</groupId>
+            <artifactId>kafka-encryption-keyrepository</artifactId>
+        </dependency>
+
```
Comment (seneque) [inline samples/generatedkey-sample/pom.xml:54]: You are in "generatedkey-sample", please don't mix the samples.

**Comment by seneque** [comment]
You have add the sample for key repository in the module "generatedkey-sample". The goal was to have separate samples for generatedkey and keyrepository

File: samples/generatedkey-sample/pom.xml (hunk 1 above)
Comment (cedricvidal) [reply samples/generatedkey-sample/pom.xml:54]: Yes, it's a mistake, don't worry :)

**Comment by cedricvidal** [comment]
@nromanetti is gonna move it to a dedicated sample as discussed

**Pull request closed without merging**